lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js

```diff
@@ -62,6 +82,15 @@ class NoVulnerableLibrariesAudit extends Audit {
       return vulns;
     }
 
+    try {
```

Comment:
idea: instead of try/catch we could make another call to `semver.valid()` . i don't care much either way. you?

Reply:
well I wanted to reuse the error object it generates to beacon to sentry, but I suppose not the end of the world to create our own with `captureMessage` instead?

I'm don't feel particularly strongly either way